### PR TITLE
8302525: Write a test to check various components send Events while mouse and key are used simultaneously

### PR DESCRIPTION
**Pull request by ravigupta30**
Main aim of this testcase is to test the behavior of various events under stress.

This testcase verifies the following assertions
a. This TestCase performs various mouse and key operations on components and checks whether all the events are getting triggered properly.
b. It also checks some of the component specific events like item, action wherever necessary.

Testing:
Tested using Mach5(20 times per platform) in macos,linux and windows and got all pass.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8302525](https://bugs.openjdk.org/browse/JDK-8302525): Write a test to check various components send Events while mouse and key are used simultaneously

### Reviewers
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**) ⚠️ Review applies to [07fb520a](https://git.openjdk.org/jdk/pull/12571/files/07fb520a38e917f5b2b63efb171dd000d7b95570)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12571/head:pull/12571` \
`$ git checkout pull/12571`

Update a local copy of the PR: \
`$ git checkout pull/12571` \
`$ git pull https://git.openjdk.org/jdk pull/12571/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12571`

View PR using the GUI difftool: \
`$ git pr show -t 12571`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12571.diff">https://git.openjdk.org/jdk/pull/12571.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/awt/event/StressTest/MouseAndKeyEventStressTest.java [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+import java.awt.event.KeyAdapter;
+import java.awt.event.KeyEvent;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+/*
+ * @test
+ * @key headful
+ * @bug 8302525
+ * @summary Test performs various mouse and key operations to checks events are getting triggered properly.
```
Comment (manukumarvs) [inline test/jdk/java/awt/event/StressTest/MouseAndKeyEventStressTest.java:47]: spelling - 'to check'

**Comment by ravigupta30** [comment]
Any volunteers for a review?

**Review by mrserb — APPROVED** [review]

**Comment by ravigupta30** [comment]
@mrserb requesting you to please sponsor.

**Pull request closed without merging**